features/mbedtls/importer/adjust-config.sh

```diff
@@ -140,6 +140,8 @@ conf unset MBEDTLS_SSL_TRUNCATED_HMAC
 
 conf unset MBEDTLS_PLATFORM_TIME_TYPE_MACRO
 
+conf set MBEDTLS_MPI_MAX_SIZE     512
```

Comment:
May be worth adding a comment on effects of this limitation, as well as its benefits.

Reply:
I added a comment, as asked